tests/ConfigurationSchemaGenerator.Tests/ConfigurationSchemaGenerator.Tests.csproj

```diff
@@ -18,10 +18,19 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" />
     <PackageReference Include="Newtonsoft.Json" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net9.0'">
+    <PackageReference Include="Microsoft.Extensions.Hosting" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'">
+    <PackageReference Include="Microsoft.Extensions.Hosting" VersionOverride="9.0.0-rc.2.24473.5" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" VersionOverride="9.0.0-rc.2.24473.5" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" VersionOverride="9.0.0-rc.2.24473.5" />
+  </ItemGroup>
```

Comment:
This should be fixed now. You should be able to revert these changes because of the following:

https://github.com/dotnet/aspire/blob/2e60ee60398402a51f15e17d1115587ed523510e/eng/Versions.props#L76-L79

This will automatically reference the 9.0 hosting packages when targeting `net9.0`.

Reply:
I've fixed this with the latest changes.